monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+    legendValues=["mean", "max", "min"],
+)
+
+disk_io_util = mongodb_timeseries(
+    "Disk IO Utilization (all instances)",
+    [
+        Target(
+            expr=f'sum by(pod) (rate(container_blkio_device_usage_total{{operation="Total", namespace="{NAMESPACE}", pod=~".*mongodb.*"}}[$__rate_interval]))',
+            legendFormat="{{pod}}",
+        )
+    ],
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+    unit="binBps",
+)
+
+# WiredTiger Panels
+eviction_thread_created = mongodb_timeseries(
+    "Eviction thread created",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_eviction_worker_thread_created{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit=UNITS.OPS_PER_SEC,
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+eviction_threads_removed = mongodb_timeseries(
+    "Eviction threads removed",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_eviction_worker_thread_removed{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit=UNITS.OPS_PER_SEC,
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+num_eviction_threads = mongodb_timeseries(
+    "Number of active eviction threads",
+    [
+        Target(
+            expr=f'mongodb_ss_wt_cache_eviction_worker_thread_active{{rs_nm=~"$rs_nm", namespace="{NAMESPACE}"}}',
+            legendFormat="{{ job }}",
+        )
+    ],
+    fillOpacity=0,
+    showPoints="auto",
+    unit="threads",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+wt_concurrency_tickets = mongodb_timeseries(
+    "Available Concurrency Tickets",
+    [
+        Target(
+            expr=f'avg(rate(mongodb_mongod_wiredtiger_concurrent_transactions_total_tickets{{txn_rw="{txn_type}", namespace="{NAMESPACE}"}}[$__rate_interval]) - rate(mongodb_mongod_wiredtiger_concurrent_transactions_out_tickets{{txn_rw="{txn_type}", namespace="{NAMESPACE}"}}[$__rate_interval]))',
+            legendFormat=txn_type,
+        )
+        for txn_type in ["read", "write"]
+    ],
+    unit="none",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=0,
+    description='A WT "ticket" is assigned out for every operation running simultaneously in the WT storage engine.',
+)
+
+write_concurrency_tickets = mongodb_timeseries(
+    "Write Concurrency Tickets",
+    [
+        Target(
+            expr=f'sum by (txn_rw) (mongodb_mongod_wiredtiger_concurrent_transactions_available_tickets{{job=~"{Jobs.shard_pattern}", txn_rw="write"}})',
+            legendFormat="{{txn_rw}}",
+        )
```

Comment:
should be "by pod" ?

```suggestion
            expr=f'sum by (pod) (mongodb_mongod_wiredtiger_concurrent_transactions_available_tickets{{job=~"{Jobs.shard_pattern}", txn_rw="write"}})',
            legendFormat="{{pod}}",
        )
```